src/components/StatTrack/styles.styl

```diff
@@ -0,0 +1,46 @@
+//
```

Comment:
Looks like there's a bit of unused stuff in here? Can probably do with a bit of a clean up.

To be honest, this component probably doesn't even need (m)any styles. If the style of the text needs to change where it is, a nicer way of doing that is to pass through a `className` prop onto the first element. Look at how FancyImage is used for an example.

It looks like you've already got a bit of code for that - if you make the div `<div className={className}>`, that'll let the parent specify styles. Then you'll need to make sure that when you use `<StatTrack>` you specify a proper className. I'll comment elsewhere about that